src/execution_plan/ops/op_node_by_label_scan.c

```diff
@@ -121,9 +174,13 @@ static Record NodeByLabelScanConsume(OpBase *opBase) {
 	return r;
 }
 
+static Record NodeByLabelScanNoOp(OpBase *opBase) {
```

Comment:
Add an explanatory comment to describe the need for this function.

Reply:
done